src/components/IconButton/styles/IconButton.module.css

```diff
@@ -9,8 +9,16 @@
     cursor: pointer;
     padding: 0;
     vertical-align: middle;
+    border-radius: 4px;
+    width: 22px;
+    height: 22px;
+    margin-left: 2px;
 }
 
 .iconButton:focus {
     outline: none;
 }
+
+.iconButton:hover {
+    background-color: rgba(0, 0, 0, 0.12);
```

Comment:
I suggest two changes:
- `background-color: rgba(77, 182, 172, 0.4);` (`teal300` @ 40%).
- `color: var(--colors-grey900);` to darken the icon on hover.